cli/commands/build.js

```diff
@@ -1,12 +1,24 @@
 const fs = require("fs");
+
 const path = require("path");
 const { format } = require("prettier");
 const { generator } = require("../../core/generator");
 
+const flatten = arr => arr.reduce((acc, val) => acc.concat(val), []);
```

Comment:
We could maybe extract these functions into a utils file, to make them testable individually. What do you think?

Reply:
I agree, I suggest something like

```
utils/
  fs
  collections
  network
  ...
```

Reply:
I like it. 👍